docs/modules/reference/pages/Glossary.adoc

```diff
@@ -1,29 +1,25 @@
 = Glossary
+Active User:::: A user who has activated access to a Payara Cloud subscription.
 
-Active User:: A user with activated access to Payara Cloud subscription.
+Administrator:: A user granted full administrative access to the applications within a subscription.
 
-Administrator:: A user with full administrative access to applications of a subscription.
+Billing Manager:: A user granted full access to the billing and payment data of a subscription.
 
-Billing Manager:: A user with full access to billing and payment data of a subscription.
+Deploy:: The act of launching your uploaded and configured application.
 
-Deploy:: The act of starting your uploaded and configured application
+DNS (Domain Name System):: A hierarchical and decentralized naming system for computers, services, or other resources connected to the Internet or a private network.
 
-DNS:: Domain Name System
+Member:: A user with limited access to a Payara Cloud subscription.
 
-Member:: A user with restricted access to Payara Cloud subscritpion
+Namespace:: A grouping or isolation of resources within a Payara Cloud subscription. A subscription can have multiple namespaces, and each namespace can host multiple applications.
 
-Namespace:: an isolation/grouping of resources.
-A Subscription can have multiple namespaces.
-A namespace can have multiple applications.
```

Comment:
It is a recommended practice to write one sentence per line in asciidoc. It helps making the diffs clear, and sentences short.